cdap-distributions/bin/build_docs_bucket.sh

```diff
@@ -44,7 +44,7 @@ TARGET_DIR=${DOCS_HOME}/target
 
 S3_BUCKET=${S3_BUCKET:-docs.cask.co}
 S3_REPO_PATH=${S3_REPO_PATH:-cdap} # No leading or trailing slashes
-VERSION=${VERSION:-5.1.2}
+VERSION=${VERSION:-5.1.3-SNAPSHOT}
```

Comment:
should just be 5.1.3 I think

Reply:
On develop this is 6.0.0-SNAPSHOT, but the other two places is still 5.1.0, @dereklwood can you confirm which version we should use for the scripts?

Reply:
This is 6.0.0-SNAPSHOT on develop, so I will keep the snapshot here.

Reply:
I think this is the docs version, which should match the CDAP version (with SNAPSHOT), and shows up here:  https://docs.cdap.io/cdap/.  I think keeping the SNAPSHOT to stay consistent with CDAP  should be fine.

I suspect this script is not used anymore (not 100% sure though).  Bamboo Docs Deploy build uses the similar [docs_bucket_current_develop.sh](https://github.com/cdapio/cdap/blob/develop/cdap-distributions/bin/docs_bucket_current_develop.sh) which has logic to determine the version

Reply:
nvm, it is apparently still used in Docs Build and Test, Docs Repo stage